### ISSUE TITLE
Webhook  "subprotocol"  identifier is missing from section 5.3.4.2. and A.3.

### ISSUE THREAD

**Issue by mlagally** [w3c/wot-thing-description#1352]
The Webhook example in section A.3 does not have a "webhook" subprotocol identifer.

Also the list of subprotocols in "5.3.4.2 Form" should contain "webhook":

For example, for HTTP and Events, it indicates which of several available mechanisms should be used for asynchronous notifications such as long polling ( longpoll ), WebSub [ websub ] ( websub ), Server-Sent Events ( sse ) [ html ] (also known as EventSource).

**Comment by sebastiankb** [comment]
good point. The question is which specific name should be used for the subprotocol. "webhook" is not a specific protocol is more a concept which needs to be implemented by a specific mechanism. [WebSub](https://www.w3.org/TR/websub) is a sample implementation.

**Comment by sebastiankb** [comment]
What comes to my mind if we should introduce a generic webhook value like "webhook_generic" to enable to exchange information that a webhook based method is used. Or/And to give a pattern like "webhook_<api_name>" to be more specific if you follow a specif api implementation "webhook_wordpress" or webhook_github".

I know it is not perfect since the name values are not standardized. However, it provides the consumer a hint what kind of implementation you follow.

What do you think?

**Comment by sebastiankb** [comment]
from today's TD call:
- it seems not useful to have this specific values in the TD
- typically, it needs to have a clear understanding by a API how the webhook is implemented
- it should be not a topic for WoT how the values of the subprotocol should look like. IANA for subprotocols have a sample list which can be used
